.gitignore

```diff
@@ -1,5 +1,6 @@
 output
 public
+Gemfile.lock
```

Comment:
why gitignore the lock? we are only using a gem here, not writing one, iirc it should be commited, but I might have forgot the best practices :)

Reply:
I used to upload it until I saw that they did not do it in miq ^^. Anyway the version is set. What do you think ? Should I upload the lock?